lib/src/adapter/upnp_adapter.dart

```diff
@@ -31,7 +31,9 @@ class UPnPAdapter extends CastAdapter {
     await disc.start(ipv6: false);
```

Comment:
@Butzlabben happy to, but does seem to be raising cast error exception:

```
_CastError (type '() => Null' is not a subtype of type '(() => FutureOr<HttpClientRequest>)?' in type cast)
```

https://github.com/Butzlabben/upnp.dart/blob/main/lib/src/service.dart#L57